turbopack/crates/turbo-tasks-macros/src/lib.rs

```diff
@@ -47,6 +47,11 @@ pub fn derive_task_input(input: TokenStream) -> TokenStream {
     derive::derive_task_input(input)
 }
 
+#[proc_macro_derive(KeyValuePair)]
+pub fn derive_key_value_pair(input: TokenStream) -> TokenStream {
+    derive::derive_key_value_pair(input)
+}
+
```

Comment:
This macro's behavior isn't obvious (it depends on field names), so this should really have some documentation.